File: ResourceManager/Models/Configurations/PasswordTagConfiguration.cs
using Microsoft.EntityFrameworkCore.Metadata.Builders;
using Microsoft.EntityFrameworkCore;
using System.Collections;
using System.Collections.Generic;
using ResourceManager.Configuration;

namespace ResourceManager.Models.Configurations
{
    public class PasswordTagConfiguration : IEntityTypeConfiguration<PasswordTag>
    {
        public void Configure(EntityTypeBuilder<PasswordTag> builder)
        {
            builder.ToTable("PasswordTags");
            builder.HasKey(x => new { x.PasswordID, x.TagID });

            builder.HasOne(x => x.Password)
                .WithMany(x => x.PasswordTags)
                .HasForeignKey(x => x.PasswordID);

            builder.HasOne(x => x.Tag)
                .WithMany(x => x.PasswordTags)
                .HasForeignKey(x => x.TagID);

        }
    }
}

File: ResourceManager/Models/Configurations/PasswordTagConfiguration.vb
Imports Microsoft.EntityFrameworkCore.Metadata.Builders
Imports Microsoft.EntityFrameworkCore

Namespace ResourceManager.Models.Configurations
    Public Class PasswordTagConfiguration
        Implements IEntityTypeConfiguration(Of PasswordTag)
        Public Sub Configure(ByVal builder As EntityTypeBuilder(Of PasswordTag)) Implements IEntityTypeConfiguration(Of PasswordTag).Configure
            builder.ToTable("PasswordTags")
            builder.HasKey(Function(x) New With {x.PasswordID, x.TagID
            })

            builder.HasOne(Function(x) x.Password).WithMany(Function(x) x.PasswordTags).HasForeignKey(Function(x) x.PasswordID)

            builder.HasOne(Function(x) x.Tag).WithMany(Function(x) x.PasswordTags).HasForeignKey(Function(x) x.TagID)

        End Sub
    End Class
End Namespace
